cviewd.8

```diff
@@ -33,11 +33,11 @@ from standard input (the network connection).  The valid commands are
 described in
 .BR cview (1).
 .SH FILES
-.B /var/athena/larvnet.cgroups
+.B /var/lib/larvnet/larvnet.cgroups
 .br
-.B /var/athena/larvnet.clusters
+.B /var/lib/larvnet/larvnet.clusters
 .br
-.B /var/athena/larvnet.printers
+.B /var/lib/larvnet/larvnet.printers
```

Comment:
These paths seem okay to my reading of the FHS.  But do we need to change them anywhere other than the man page (or is that in a different package)?